doc/mail.md

```diff
@@ -0,0 +1,28 @@
+# Email addresses
+
+The `@nixos.org` email server is currently hosted on [ImprovMX](https://improvmx.com/).
+
+ImprovMX is configured to forward emails on the following addresses to the respective users:
+- community: moderation@nixos.org
+- finance: [@edolstra](https://github.com/edolstra), [kate](https://discourse.nixos.org/u/kate) (Discourse)
+- foundation: [@edolstra](https://github.com/edolstra), [@refroni](https://github.com/refroni), [kate](https://discourse.nixos.org/u/kate) (Discourse)
+- fundraising: [@fricklerhandwerk](https://github.com/fricklerhandwerk)
+- infra: [@mweinelt](https://github.com/mweinelt), [@zimbatm](https://github.com/zimbatm), [@vcunat](https://github.com/vcunat), [@edef1c](https://github.com/edef1c), [@Mic92](https://github.com/Mic92)
+- marketing: [@idabzo](https://github.com/idabzo), Patrick Callahan (?), [@Kranzes](https://github.com/Kranzes) [@tomberek](https://github.com/tomberek), [@garbas](https://github.com/garbas)
```

Comment:
There's some email for a Patrick Callahan, but I could not find any matching user handle anywhere..?

Reply:
This is `@pxc` on discourse, @therealpxc on GitHub.